label_sync/labels.css

```diff
@@ -0,0 +1,681 @@
+.committee\00002fconduct {
```

Comment:
one more though, how do we plan to get this into deck? maybe we should generate the label css into `prow/cmd/deck/static/labels.css` and update the frontend to consume it?

Reply:
Yeah for deck's consuming this should be in `deck/static`, but do we need to keep a copy of it here also?

Reply:
It seems fine to generate it there instead. I don't think anything else needs to consume it yet

Reply:
Okay I will update the path

Reply:
thanks! :-)